src/main/java/gui/modal/CreationModalView.java

```diff
@@ -81,73 +129,174 @@ public CreationModalView(RootPane rootPane, int numberOfCamerasInTimeline,
-
-        initNameDescriptionFields();
-        initCountFields();
+        // force minimum size
+        getModalStage().setMinWidth(width);
+        getModalStage().setMinHeight(height);
+
+        // Create a new VBox for vertical layout
+        this.viewPane = new VBox();
+
+        // Add label at top
+        initTitleLabel();
+
+        // add space for textfields and checkboxes
+        this.contentPane = new HBox();
+        this.contentPane.setAlignment(Pos.CENTER);
+        this.contentPane.setPadding(new Insets(0, 20, 0, 0));
+        this.contentPane.setPrefHeight(10000);
+        this.contentPane.setSpacing(40.0);
+        this.contentPane.setStyle(centerStyle);
+        this.viewPane.getChildren().add(contentPane);
+
+        // actually add textfields and checkboxes
+        initTextFields();
         initCamCheckBoxes();
 
-        creationButton = new Button("Create");
-        creationButton.setOnMouseClicked(this::createShot);
-        this.viewPane.getChildren().add(creationButton);
+        // add buttons at bottom.
+        initButtons();
+
+        // once we're done, setup shadows etc.
+        initEffects();
 
         super.setModalView(this.viewPane);
         super.displayModal();
     }
 
     /**
-     * Initialize the shot name and description fields.
+     * Initialize title label.
+     */
+    private void initTitleLabel() {
+        titleLabel = new Label("Add a new shot...");
+        titleLabel.setStyle(topStyle);
+        titleLabel.setAlignment(Pos.CENTER_LEFT);
+        titleLabel.setPadding(new Insets(0, 0, 0, 20));
+        titleLabel.setPrefWidth(10000);
+        titleLabel.setPrefHeight(10000);
+        this.viewPane.getChildren().add(titleLabel);
+    }
+
+    /**
+     * Sets up effects and adds them to the appropriate panes.
+     */
+    private void initEffects() {
+        topInnerShadow = new InnerShadow(BlurType.GAUSSIAN, Color.rgb(0, 0, 0, 0.15),
+                1, 1, 0, -2);
+        topOuterShadow = new InnerShadow(BlurType.GAUSSIAN, Color.rgb(0, 0, 0, 0.05),
+                15, 0.2, 0, 1);
+        bottomOuterShadow = new DropShadow(BlurType.GAUSSIAN, Color.rgb(0, 0, 0, 0.05),
+                15, 0.2, 0, -1);
+        titleLabel.setEffect(topInnerShadow);
+        contentPane.setEffect(topOuterShadow);
+        buttonPane.setEffect(bottomOuterShadow);
+    }
+
+    /**
+     * Initializes pane with buttons at bottom.
+     */
+    private void initButtons() {
+        // setup button pane
+        this.buttonPane = new HBox();
+        this.buttonPane.setSpacing(20.0);
+        this.buttonPane.setAlignment(Pos.CENTER_LEFT);
+        this.buttonPane.setPrefHeight(10000);
+        this.buttonPane.setStyle(bottomStyle);
+        this.buttonPane.setPadding(new Insets(0, 0, 0, 20));
+        this.viewPane.getChildren().add(buttonPane);
+
+        // Add cancel button
+        cancelButton = new StyledButton("Cancel");
+        cancelButton.setOnMouseReleased(e -> {
```

Comment:
The team disagrees on putting the validation in the controller, the button pressed listener is agreed to be moved to a controller